mmv1/products/oracledatabase/AutonomousDatabase.yaml

```diff
@@ -0,0 +1,622 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+name: 'AutonomousDatabase'
+description: An AutonomousDatabase resource.
+references:
+  guides:
+    'Create Autonomous databases': 'https://cloud.google.com/oracle/database/docs/create-databases'
+  api: 'https://cloud.google.com/oracle/database/docs/reference/rest/v1/projects.locations.autonomousDatabases'
+docs:
+id_format: 'projects/{{project}}/locations/{{location}}/autonomousDatabases/{{autonomous_database_id}}'
+base_url: 'projects/{{project}}/locations/{{location}}/autonomousDatabases'
+self_link: 'projects/{{project}}/locations/{{location}}/autonomousDatabases/{{autonomous_database_id}}'
+create_url: 'projects/{{project}}/locations/{{location}}/autonomousDatabases?autonomousDatabaseId={{autonomous_database_id}}'
+immutable: true
+timeouts:
+  insert_minutes: 240
+  update_minutes: 120
+  delete_minutes: 120
+import_format:
+  - 'projects/{{project}}/locations/{{location}}/autonomousDatabases/{{autonomous_database_id}}'
+autogen_async: true
+async:
+  actions: ['create', 'delete', 'update']
+  type: 'OpAsync'
+  operation:
+    base_url: '{{op_id}}'
+    path: 'name'
+    wait_ms: 1000
+    timeouts:
+      insert_minutes: 240
+      update_minutes: 120
+      delete_minutes: 120
+  result:
+    path: 'response'
+    resource_inside_response: true
+  error:
+    path: 'error'
+    message: 'message'
+custom_code:
+examples:
```

Comment:
Can we include an example/test case that doesn't set `properties`? We'd be fine to omit it from docs if you'd prefer.

Reply:
Sorry, I couldn't understand it. If we didn't set the properties the create API will fail as it has required field. And we definitely prefer not to have that in docs.